app/code/Magento/AdminNotification/Test/Unit/Model/System/Message/CacheOutdatedTest.php

```diff
@@ -3,44 +3,60 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+declare(strict_types=1);
```

Comment:
Please revert such additions, it's better to deliver them separately and do not mix with other changes.